src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+img {
+  /* display: flex; */
+  margin: 10px 0 10px 50px;
+  cursor: pointer;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+
+  color: black;
+  text-decoration: none;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  position: relative;
+  padding: 23px 0;
+
+  color: black;
+  text-decoration: none;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  content: "";
+  display: block;
+  height: 4px;
+  width: 37px;
+  border-radius: 8px;
+  background-color: #00acdc;
+
+}
+
+.video {
+  margin-right: -20px;
```

Comment:
why do you need negative margin?

Reply:
fix it https://github.com/mate-academy/layout_moyo-header/pull/2442/files#r1056818198